Source/Core/Common/ChunkFile.h

```diff
@@ -229,6 +229,19 @@ class PointerWrap
     return current;
   }
 
+  // The caller needs to fill in val with the appropriate value later on.
+  [[nodiscard]] u8* GetPositionAndDoInt(u32& val)
+  {
+    u8* prevPointer = *m_ptr_current;
+    Do(val);
+    return prevPointer;
+  }
+
+  u32 GetOffsetFromPreviousPosition(u8* prevPointer)
+  {
+    return (u32)((*m_ptr_current) - prevPointer) - sizeof(u32);
+  }
```

Comment:
This function doesn't make it clear that it's skipping over a `u32`. You're probably better off just making this a more generic

```
  size_t GetOffsetFromPreviousPosition(u8* previous_position)
  {
    return static_cast<size_t>((*m_ptr_current) - previous_position);
  }
```